client/interfaces.go

```diff
@@ -278,6 +278,9 @@ type ImageCreateArgs struct {
 	// Filename for the rootfs file
 	RootfsName string
 
+	// Image creation only for the image synchronization
+	SyncOnly bool
```

Comment:
I think you don't need this additional request flag. We already have an HTTP header that we use to tag requests that are coming from within nodes of the LXD cluster itself, rather than from the user. If you use the cluster.Connect() helper, like you do, then this header is set automatically for you, and you can use isClusterNotification(r) to check if it's present, and you can replace the ```shared.IsTrue(r.Header.Get("X-LXD-image-sync"))``` check.